symbols/gnetman/Makefile.am

```diff
@@ -0,0 +1,18 @@
+instdirname = sym-gnetman
+datasymdir = $(GEDADATADIR)/$(instdirname)
+
+dist-hook:
+	$(MKDIR_P) $(distdir)
+	cp -p $(srcdir)/*.[sS][yY][mM] $(distdir)
+
+install-data-local:
+	$(MKDIR_P) $(DESTDIR)$(datasymdir)
+	$(INSTALL_DATA) $(srcdir)/*.[sS][yY][mM] $(DESTDIR)$(datasymdir)
+
+uninstall-local:
+	rm -fr $(DESTDIR)$(datasymdir)
+
+MOSTLYCLEANFILES = *.log *~
```

Comment:
I would not use those patterns for `*CLEANFILES` from pretty old makefiles, all those log, core and backup files. Please use the names you really found are there in most situations.

Reply:
IIUC, only `MAINTAINERCLEANFILES = Makefile.in` line should be in 
`{gnetman,verilog,vhdl}/Makefile.am`, right?

Reply:
However, according to `automake` [manual](https://www.gnu.org/software/automake/manual/html_node/Clean.html)
```
maintainer-clean should not delete anything that needs
to exist in order to run ‘./configure && make’
```

I see `MAINTAINERCLEANFILES = Makefile.in` in many places,
but `./configure` fails if it cannot find `Makefile.in`:
```
config.status: error: cannot find input file: `symbols/gnetman/Makefile.in'
```
So, is it a bug in `Makefile.am` files?

Reply:
AFAIR, I deleted whole `MAINTAINERCLEANFILES` lines before (along with `Makefile.in`) and nothing changed. Maybe it was needed some time ago for autotools, though nowadays I don't see any reason to leave it as is.

Reply:
OK, `*CLEAN*` lines removed.